vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,27 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+// HealthcheckResult stores information about a single run of a healthcheck probe
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting, -1=error running probe
+	Output     string // Output from last check
```

Comment:
If so, we can raise the limits later when we add that. Allowing containers to store unbounded amounts of data in Docker's JSON status files (and return it in inspect replies) would make DoS attacks very easy. Allowing binary data prevents tools from displaying the output to the user in a readable form. It would be better to add a separate field later if we decide on some structured payload format (and not mix it with error output).